test/govspeak_test.rb

```diff
@@ -341,8 +341,6 @@ class GovspeakTest < Minitest::Test
 
   # Regression test - the surrounded_by helper doesn't require the closing x
   # so 'xaa' was getting picked up by the external link helper above
-  # TODO: review whether we should require closing symbols for these extensions
```

Comment:
should we do the todo?

Reply:
Probably a bit late now as the extensions without closing symbols will be scattered throughout content.